ts/utils/appVersion.ts

```diff
@@ -64,17 +65,18 @@ export const getAppVersion = () =>
 /**
  * return true if the app must be updated
  * @param serverInfo the backend info
+ * @param section
  */
 export const isUpdateNeeded = (
-  serverInfo: ServerInfo | undefined,
+  serverInfo: IOVersionInfo | undefined,
```

Comment:
```suggestion
  versionInfo: IOVersionInfo | undefined,
```

Reply:
I'm working on renaming and refactoring in https://pagopa.atlassian.net/browse/IAI-122, if isn't a problem I would concentrate all the renaming part there